src/main/drivers/system_stm32f4xx.c

```diff
@@ -39,18 +39,39 @@ void systemReset(void)
     NVIC_SystemReset();
 }
 
+PERSISTENT uint32_t bootloaderRequest = 0;
+
 void systemResetToBootloader(void)
 {
     if (mpuResetFn) {
         mpuResetFn();
     }
 
-    *((uint32_t *)0x2001FFFC) = 0xDEADBEEF; // 128KB SRAM STM32F4XX
+    bootloaderRequest = 1;
```

Comment:
More magic is actually useful in this case when debugging, you are communicating through reset ... and `1` is boring

Reply:
I can replace it by a constant `BOOTLOADER_REQUEST_MAGIC_COOKIE = 0xDEADBEEF` if you really want it :-)